frontend/src/components/House/House.js

```diff
@@ -4,18 +4,25 @@ import styles from './styles';
 
 export default class House extends Component {
   onNewIssue = () => {
-    this.props.onNewIssue( this.props.data);
```

Comment:
I have fixed some stuff like this in my review changes, also moved inline styles to aphrodite. I have also refactored this component to use a Card using your css ... merge will be fun :)

Reply:
I will wait for your PR to be merged first then.

Reply:
unfortunately my refactor is still two pr's away. Don't worry I'll sort it ... I do have more time than you after all ;)